core/src/test/scala/unit/kafka/security/authorizer/AuthorizerTest.scala

```diff
@@ -142,16 +161,14 @@ class AuthorizerTest extends QuorumTestHarness with BaseAuthorizerTest {
 
   // Authorizing the empty resource is not supported because we create a znode with the resource name.
   @ParameterizedTest
-  @ValueSource(strings = Array(KRAFT, ZK))
+  @ValueSource(strings = Array(KRAFT))
```

Comment:
Also update the comment above that mentions `znode`

Reply:
Updated it.